grafana/dashboards/health.json

```diff
@@ -694,7 +694,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "tcp_close_total{deployment=\"$deployment\", namespace=\"$namespace\", direction=\"inbound\",classification=\"failure\"}",
+              "expr": "tcp_close_total{deployment=\"$deployment\", namespace=\"$namespace\", direction=\"inbound\",errno=\"\"}",
```

Comment:
Ah, hmm, this chart is a bit off:

![image](https://user-images.githubusercontent.com/9226/47877413-11bc6080-ddd9-11e8-8a00-404684fd9088.png)

In order to display failures, I think your match condition needs to be `errno!=\"\"`, not `errno=\"\"`.

And this chart doesn't include a legend, which is why the tooltip is so long. If you update this chart to include a legendFormat attr on line 700, that should fix it.

```diff
diff --git a/grafana/dashboards/health.json b/grafana/dashboards/health.json
index ed22d43c..2cf904cb 100644
--- a/grafana/dashboards/health.json
+++ b/grafana/dashboards/health.json
@@ -694,9 +694,10 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "tcp_close_total{deployment=\"$deployment\", namespace=\"$namespace\", direction=\"inbound\",errno=\"\"}",
+              "expr": "tcp_close_total{deployment=\"$deployment\", namespace=\"$namespace\", direction=\"inbound\",errno!=\"\"}",
               "format": "time_series",
               "intervalFactor": 1,
+              "legendFormat": "{{peer}} {{errno}}",
               "refId": "A"
             }
           ],
```